src/liquid/components/ld-select/ld-select.tsx

```diff
@@ -363,17 +366,25 @@ export class LdSelect {
     }
 
     if (!initialized) {
-      this.internalOptionsHTML = this.el.innerHTML['replaceAll'](
-        /<ld-option/g,
-        `<ld-option-internal${this.multiple ? ' mode="checkbox"' : ''}${
+      let internalOptionsHTML = ''
```

Comment:
What is the difference in Output compared to the previous approach?

Reply:
The checked props were missing on the internal options with the previous solution. Now they are not missing.

Reply:
Shouldn't this result in a changed test somehow?

Reply:
It doesn't. I can't figure out why this makes a difference for the docs-pick-theme component but no difference in the tests.